atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/Event.java

```diff
@@ -48,6 +50,7 @@ static Event fromKeywordMap(Map<Keyword, ?> map) {
     static Map<Keyword, Object> toKeywordMap(Event event) {
         Map<String, Object> rawStringMap = OBJECT_MAPPER.convertValue(event, new TypeReference<Map<String, ?>>() {});
         return EntryStream.of(rawStringMap)
+                .filterValues(Objects::nonNull)
```

Comment:
Hmm, my concern with this is that `nulls` will be silently filtered out, and we could accidentally filter out some things. Maybe this won't be a problem though.

Presumably if an event has a required field, there will be an error somewhere during construction? E.g. I try to convert an object `{:type :ok :value nil}`? Could we add a `can/cannotDeserialiseOkReadWithNullValue` test to capture the behaviour?